src/lib/list/selection-list.spec.ts

```diff
@@ -0,0 +1,370 @@
+import {async, TestBed, ComponentFixture, inject} from '@angular/core/testing';
+import {Component, DebugElement} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {MdSelectionList, MdListOption, MdListModule} from './index';
+import {createKeyboardEvent} from '@angular/cdk/testing';
+import {UP_ARROW, DOWN_ARROW, SPACE} from '../core/keyboard/keycodes';
+import {Platform} from '../core/platform/index';
+
+describe('test normal selection list with list option', () => {
```

Comment:
You can wrap all of these child `describe` blocks in one top-level block:
```ts
describe('MdSelectionList', () => {
  ...
});
```

Then, each describe block has a name that modifies the selection list, e.g.

```ts
describe('MdSelectionList', () => {
  describe('with multiple options', () => {
    ...
  });

  describe('with one option', () => {
    ...
  });

  describe('with disabled state', () => {
    ...
  });
});
```

This will make the test failures report like 
```
MdSelectionList with one option should select an option
```